literature.bib

```diff
@@ -6657,6 +6916,24 @@ @inproceedings{FMS+:SANER17
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2016 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@inproceedings{GZH+:ECCV16,
+  title        = {{MS-Celeb-1M: A Dataset and Benchmark for Large-Scale Face Recognition}},
+  author       = {Guo, Yandong and Zhang, Lei and Hu, Yuxiao and He, Xiaodong and Gao, Jianfeng},
+  booktitle    = {Computer Vision -- ECCV 2016},
+  pages        = {87--102},
+  year         = 2016,
+  organization = Springer
+}
+
+@inproceedings{KFB:VSTTE16,
```

Comment:
we already have this entry as KFB:16, although that entry looks kind of messed up. maybe both can be combined

Reply:
Could not find KFB:16. Is this already on the main branch?

Reply:
good catch, that's only in my local version